adcircpy/mesh/base.py

```diff
@@ -116,6 +116,12 @@ def to_dict(self):
         nodes = {nid: (coo, val) for nid, coo, val in zip(self._id, self._coords, self.values)}
         return nodes
 
+    def __eq__(self, other: 'Nodes') -> bool:
+        try:
+            return np.all(self.coords == other.coords) and np.all(self.values == other.values)
```

Comment:
should this be to a certain level of precision?

Reply:
I'd say that even the slightest precision differences will makes the mesh categorically different.